src/DatePicker/style.css

```diff
@@ -0,0 +1,200 @@
+    .ant-picker-cell-inner::before {
+      @apply rounded-full border-neutral-300;
+    }
+  }
+
+  &-cell .ant-picker-cell-inner {
+    @apply rounded-full inline-block w-auto;
+    @apply px-0;
+  }
+
+  &-year-panel .ant-picker-cell-inner {
+    @apply px-2.5 rounded-lg;
+  }
+
+  &-cell-in-view.ant-picker-cell-selected {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single)::before,
+  &-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single)::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range {
+    @apply text-white;
+  }
+
+  &-header-view button {
+    @apply text-slate-700 text-base font-inter font-bold uppercase leading-6;
+    @apply hover:opacity-80 hover:text-slate-700;
+  }
+
+  &-header-view button:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range-end {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range {
+    @apply hover:bg-slate-100;
+
+    .ant-picker-cell-range-hover {
+      @apply hover:bg-red-300;
+      @apply bg-pink-600;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range.ant-picker-cell-range-hover::before {
+    @apply bg-slate-400;
+    @apply hover:bg-pink-200;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single).ant-picker-cell-range-hover-start::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single):not(.ant-picker-cell-range-end)
+    .ant-picker-cell-inner {
+    @apply rounded-tl-2xl rounded-bl-2xl;
+  }
+
+  &-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single):not(.ant-picker-cell-range-start)
+    .ant-picker-cell-inner {
+    @apply rounded-tr-2xl rounded-br-2xl;
+  }
+}
+
+.ant-form-item-label {
+  @apply pb-2 mb-0;
+}
+
+.ant-form-item .ant-form-item-label {
+  @apply leading-none;
+}
+
+.ant-form-item-has-error {
+  .ant-picker {
+    &-input > input {
+      @apply text-red-600;
+    }
+
+    &-separator {
+      @apply text-red-600;
+    }
+
+    &-range-separator {
+      @apply text-red-600;
+    }
+
+    &-suffix {
+      @apply text-red-600;
+    }
+
+    &-clear > span > svg {
+      background-image: var(--ic-x-16);
+    }
+  }
+}
+
+.ant-picker-body > table > thead > tr th:nth-child(n + 1):nth-child(-n + 8) {
+  @apply indent-[9999px] leading-[0];
+}
+
+.ant-picker-body > table > thead > tr th:nth-child(n + 1):nth-child(-n + 8):after {
+  @apply block font-normal indent-0 leading-normal;
+}
+
+/* sunday */
+.ant-picker-body > table > thead > tr th:nth-child(1):after {
```

Comment:
You don't need to have a long selector here. Please check if you can use a short selector rules and apply it to other places too.